### ISSUE TITLE
vMix Social alternative in OBS

### ISSUE THREAD

**Issue by ASchneiderBR** [noeal-dac/Animated-Lower-Thirds#5]
**Is your feature request related to a problem? Please describe.**
Not at all :)

**Describe the solution you'd like**
Hello sir! First of all, thanks for this great solution, you had delivered an incredible addon for the OBS community. I was wondering if this could be a nice foundation for an important addon only found on expensive broadcast solutions: 
vMix Social. The way you've managed to create a command panel and how elegant it is, your solution has the best path of doing it, in my opinion. I know that this would a lot of work, specially creating a safe way of users to connect to their social accounts and making it all possible just using HTML code.

**Describe alternatives you've considered**
vMix Social? haha

**Additional context**
https://www.vmix.com/products/vmix-social.aspx

I know this is asking for too much maybe, but there's nothing else available at the moment. Thanks again for your great work here, it is top notch. just brilliant :)

**Comment by noeal-dac** [comment]
Hi! Thank you for your kind words.
I would be lying if I said that I haven't thought about integrating those functions into the tool. As you say that would require some work. xD
Anyway, I find it a really interesting feature. I'll keep thinking of a solution.
Thanks for your support!

**Comment by ASchneiderBR** [comment]
Wow! Thank you so much for the response! Great to know that this idea is at least a possibility :) I truly believe that this is a game changer.

**Comment by InThyWord** [comment]
Yeah definitely please consider this as this is a game changer for OBS. For Mac and Windows too

🙂🙂🙂🙂🙂🙂
